assets/css/main.scss

```diff
@@ -152,16 +152,19 @@
     max-width: 500px;
 }
 
-.con-button button {
-    background-color: #f37726;
-    color: white;
-    margin: 0 .5em;
-    padding: 1em;
-    border-radius: 5px;
-    border: none;
-    font-size: 1em;
-    font-weight: 600;
-    cursor: pointer;
+div.con-buttons {
+    margin-top: 2em;
+    a.con-button {
```

Comment:
Is this acceptable support level https://caniuse.com/css-nesting - 87.4% of global users?

Reply:
Isn't this SCSS getting compiled to CSS by Jekyll in the actual HTML output?

Reply:
Ah, right!